.claude/skills/court-vision-design/colors_and_type.css

```
@@ -0,0 +1,298 @@
+/* ==========================================================================
+   Court Vision — Colors & Type
+   ==========================================================================
+   Lucas Lawrence's basketball-themed personal site (courtfolio).
+   Two coexisting moods:
+   - "Arena" — black/charcoal stage with orange spotlight + yellow banners
+   - "Hardwood" — warm wood-grain leather binder, cream paper, ink black
+   ========================================================================== */
+
+/* ---- Webfonts ----------------------------------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700;800&family=Geist+Mono:wght@400;500;600&family=Patrick+Hand&display=swap');
```

Stylelint (17.9.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700;800&family=Geist+Mono:wght@400;500;600&family=Patrick+Hand&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700;800&family=Geist+Mono:wght@400;500;600&family=Patrick+Hand&display=swap'" (import-notation)

(import-notation)